e2e/dractions/crud.go

```diff
@@ -46,23 +47,38 @@ func getDRPC(client client.Client, namespace, name string) (*ramen.DRPlacementCo
 	return drpc, nil
 }
 
-func createDRPC(client client.Client, drpc *ramen.DRPlacementControl) error {
+func createDRPC(ctx types.Context, client client.Client, drpc *ramen.DRPlacementControl) error {
+	log := ctx.Logger()
+
 	err := client.Create(context.Background(), drpc)
 	if err != nil {
 		if !errors.IsAlreadyExists(err) {
 			return err
 		}
-		// ctx.Log.Info("drpc " + drpc.Name + " already Exists")
+
+		log.Debugf("drpc \"%s/%s\" already exist", drpc.Namespace, drpc.Name)
 	}
 
+	// pretty-print the drpc specification
+	drpcSpecJSON, err := json.MarshalIndent(drpc.Spec, "", "  ")
+	if err != nil {
+		log.Errorf("Error marshaling drpc for logging: %v", err)
```

Comment:
We never log the error *and* return it. This will log the same error twice. This case is very unlikely so just return the error.